### ISSUE TITLE
Doesn't render in el

### ISSUE THREAD

**Issue by klaemo** [AmpersandJS/ampersand-select-view#46]
Hey,
whenever I specify an `el` the SelectView doesn't get rendered at all. I use it in conjunction with `ampersand-form-view`.
I've tried the latest version on npm and `master`, all kinds of different combinations of `autoAppend` and `autoRender` to no avail.

```javascript
new SelectView({
  el: this.el.querySelector('[data-hook=package-name]')
})
```

Without a custom `el` it works fine. My input-views in the form work fine as well (with custom `el`).
I'm not sure if it's a bug or if I'm doing something wrong, but I have no idea what to try next.

Thank you!

**Comment by cdaringe** [comment]
Hi @klaemo:

I can't look into this immediately.  I wouldn't be surprised if [this had something to do with it](https://github.com/AmpersandJS/ampersand-view/pull/115), but can't say for certain.  in the meantime, fiddle?

Thx!

**Comment by klaemo** [comment]
Thanks for your feedback :) 
That does look related. It seems as if the parent form view thinks the select-view has already been rendered.
I tried removing the check here: https://github.com/AmpersandJS/ampersand-select-view/blob/master/ampersand-select-view.js#L82 but it didn't even get called.

PS: I have fiddled, oh boy have I fiddled!!11 ;-)

**Comment by klaemo** [comment]
I found the following workaround:

```javascript
```

**Comment by klaemo** [comment]
I found the following workaround:

```javascript
const SelectViewFix = SelectView.extend({
  initialize(opts) {
    SelectView.prototype.initialize.apply(this, arguments)
    this._el = opts._el
  },
  render() {
    SelectView.prototype.render.call(this)
    if (this._el) this._el.appendChild(this.el)
    return this
  }
})
```

This prevents `rendered` from being set to `true` automatically.

**Comment by cdaringe** [comment]
apologies for late follow up.  confirmed, the above issue i suggested is the root of the issue.  simply commenting out [line 82](https://github.com/AmpersandJS/ampersand-select-view/blob/master/ampersand-select-view.js#L82) does the trick.  will try to resolve that issue in effort to close this one

**Comment by cdaringe** [comment]
@klaemo, I didn't see that you had followed up saying it was unsuccessful for you.  I'm not sure how you constructed your example, but I modified the `demo/demo.js` as simple example.  Commenting out the `rendered` check in this module and using [this gist seed as demo.js](https://gist.github.com/cdaringe/d92275d490a3edaddeb5) is all it takes.  load 'er up, and you'll see it render into your container el successfully

**Comment by klaemo** [comment]
Nope your demo doesn't work for me.
![bildschirmfoto 2015-05-13 um 11 39 58](https://cloud.githubusercontent.com/assets/778624/7607857/fb78c7a2-f964-11e4-946b-bf27ba7453dd.png)
![bildschirmfoto 2015-05-13 um 11 40 16](https://cloud.githubusercontent.com/assets/778624/7607858/019c793a-f965-11e4-955e-ae531361c04f.png)

**Comment by cdaringe** [comment]
hmm. here is the process i just ran:
- branched from upstream/master
- replaced demo.js with my gist content
- ran npm run demo

it all worked out!  can you do a double check?  gotta get that other issue ironed out

![screen shot 2015-05-21 at 6 02 52 pm](https://cloud.githubusercontent.com/assets/1003261/7760724/e7ce5c1e-ffe3-11e4-8a45-cf8e885ab9d0.png)
![screen shot 2015-05-21 at 6 03 46 pm](https://cloud.githubusercontent.com/assets/1003261/7760725/e81485b8-ffe3-11e4-9da0-aa93306b5d39.png)
![screen shot 2015-05-21 at 6 03 50 pm](https://cloud.githubusercontent.com/assets/1003261/7760726/e815ffb0-ffe3-11e4-849b-b35d808e86c0.png)
![screen shot 2015-05-21 at 6 04 41 pm](https://cloud.githubusercontent.com/assets/1003261/7760727/e8194210-ffe3-11e4-84e8-2bf67d168510.png)

**Comment by klaemo** [comment]
That's so weird! I did pretty much the same thing, but it doesn't work. I even updated to the latest &-view just to be safe.

![git status](https://cloud.githubusercontent.com/assets/778624/7771584/a6141c7c-0098-11e5-8ea0-e31c8bf5820e.png)
![changes to demo](https://cloud.githubusercontent.com/assets/778624/7771588/adfcc2a4-0098-11e5-93d5-67d8d790ab5a.png)

Sorry, I can't be of any more help :(

**Comment by klaemo** [comment]
btw, I just manually applied your fix from https://github.com/AmpersandJS/ampersand-view/pull/119 and I can confirm that it solves this issue as well. :)

**Comment by wraithgar** [comment]
@klaemo mind submitting a PR for this repo too then?

**Comment by cdaringe** [comment]
@wraithgar patching this repo probably isn't the ideal solution until a decision is made on the &view 'rendered' bug.  Would appreciate some follow up on that one!  Link above

**Comment by klaemo** [comment]
@wraithgar I think here we only need to bump the `ampersand-view` dep once a version with the fix has been published. Or am I missing something?

**Comment by cdaringe** [comment]
@klaemo, yea, you are correct.

**Comment by cdaringe** [comment]
4.3.1 released, with 8.0.0 dep!  this should do the trick!
